opentech/static_src/src/app/src/redux/actions/submissions.js

```diff
@@ -49,10 +50,31 @@ export const setCurrentSubmissionRound = id => ({
     id,
 });
 
-export const setCurrentSubmission = id => ({
-    type: SET_CURRENT_SUBMISSION,
-    id,
-});
+
+export const loadSubmissionFromURL = () => (dispatch, getState) => {
+    const urlParams = new URLSearchParams(window.location.search);
+    if (urlParams.has('submission')) {
+        const activeId = Number(urlParams.get('submission'));
+        return dispatch(setCurrentSubmission(activeId));
+    }
+    return null;
+}
+
+
+export const setCurrentSubmission = id => (dispatch, getState) => {
+    const submissionID = getCurrentSubmissionID(getState())
+
+    if (id && submissionID !== id) {
```

Comment:
`id` could be `0` I guess and would fail this test. Should we check for `undefined` or `null` instead?

Reply:
You are correct.